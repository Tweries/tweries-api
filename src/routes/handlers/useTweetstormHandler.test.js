const { matcher, mockSend, reqBase, resBase } = require('../mocks');
const useTweetstormHandler = require('./useTweetstormHandler');

test('useTweetstormHandler', async () => {
  const req = {
    ...reqBase,
    body: {
      items: ['TODO'],
      userId: 'twitter|1183836409850814464'
    }
  };

  const res = { ...resBase };

  await useTweetstormHandler(req, res);

  expect(mockSend.mock.calls[0][0]).toMatchSnapshot(matcher);
});
