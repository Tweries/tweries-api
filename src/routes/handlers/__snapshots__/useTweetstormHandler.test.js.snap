// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`useTweetstormHandler 1`] = `
Object {
  "env": "test",
  "ids": Array [],
  "items": Array [
    "TODO",
  ],
  "message": "ðŸ‘Œ",
  "name": "tweries-api",
  "timestamp": Any<Number>,
  "userId": "twitter|1183836409850814464",
  "version": "0.10.0",
}
`;
